backend/schemas/source/excel/templates/federal-awards-expended-template.jsonnet

```diff
@@ -37,6 +37,7 @@ local open_ranges_defns = [
   [Sheets.open_range, {}, 'Cluster Name', 'cluster_name'],
   [Sheets.open_range, {}, 'If State Cluster, Enter State Cluster Name', 'state_cluster_name'],
   [Sheets.open_range, {}, 'If Other Cluster, Enter Other Cluster Name', 'other_cluster_name'],
+  // 20230525 HDMS FIXME: A formula to auto calculate federal_program_total in the excel is missing (see instructions in census template)!!!
```

Comment:
There is an instruction related to this field in census FederalAwardsExpended.xlsx template but I am not sure how we are going to achive this.

Reply:
I think this has been ticketed separately at this point, so we can track it/address it. Leaving this here for now is fine as a reminder. We've decided (in conversation) there's more than one way to solve this, and none of them look scary/look like blockers. We'll leave the FIXME as a reminder in the codebase.